pulsar-storm/src/main/java/org/apache/pulsar/storm/PulsarSpout.java

```diff
@@ -454,7 +454,7 @@ public void close() throws PulsarClientException {
 
         @Override
         public void unsubscribe() throws PulsarClientException {
-            consumer.unsubscribe();
```

Comment:
It might be better to split the actual fix from the version change. eg: first the version update and then using the new API